src/main/java/org/jenkinsci/test/acceptance/po/WizardCreateAdminUser.java

```diff
@@ -73,8 +73,9 @@ public void shouldCreateTheUserSuccessfully() {
     }
 
     public void wizardShouldFinishSuccessfully() {
-        waitFor(by.button(START_BUTTON_TEXT));
-        clickButton(START_BUTTON_TEXT);
+        By installDoneButtonSelector = by.button(".btn-primary.install-done");
```

Comment:
Should this not be `by.css` as well?

Reply:
`by.button` works, but `by.css` is simpler indeed.